api/server.js

```diff
@@ -15,4 +15,8 @@ server.use(express.json());
 server.use('/api/auth', authRouter);
 server.use('/api/jokes', authenticate, jokesRouter);
 
+server.get('/', (req, res) => {
```

Comment:
A lot of APIs will have a `/status` like this, but that also checks dependencies like databases, the joke API, etc.